src/Validation/Validator.php

```diff
@@ -104,6 +105,8 @@ public function errors(array $data, $newRecord = true)
     {
         $errors = [];
 
+        $data = Hash::flatten($data);
```

Comment:
How do you see this interacting with the existing [nested validator](https://book.cakephp.org/3.0/en/core-libraries/validation.html#nesting-validators) features?

Reply:
What I wanted is pretty much to store a modelless form inside my entity.